plugins/pulp_rpm/plugins/distributors/export_distributor/generate_iso.py

```diff
@@ -87,9 +88,26 @@ def _make_iso(file_list, target_dir, output_dir, filename):
     if status != 0:
         log.error("Error creating iso %s; status code: %d; output: %s" % (file_path, status, out))
     else:
+        _add_digest_file(file_path)
         log.info('Successfully created iso %s' % file_path)
 
 
+def _add_digest_file(file_path):
+    exportedISO = open(file_path, "rb")
+    hashers = {}
+    for hasher in hashlib.algorithms_guaranteed:
+        hashers[hasher] = getattr(hashlib, hasher)()
+        while True:
```

Comment:
This whilte loop should not be inside this loop. Instead you should add another loop inside the while loop. The inner loop should update each hasher.